### PR TITLE
include the description of  FD-1094 in release note v6.4.0

### PR DESCRIPTION
**Pull request by songrijie**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
include the description of  FD-1094 in release note v6.4.0

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.4 (TiDB 6.4 versions)
- [ ] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: releases/release-6.4.0.md [hunk 1]
```diff
@@ -130,6 +130,10 @@ TiDB 版本：6.4.0-DMR
 
     更多信息，请参考[用户文档](/system-variables.md#tidb_stats_load_sync_wait-从-v540-版本开始引入)。
 
+* 降低批量写入请求对轻量交易事务写入的响应时间的影响 [#13313](https://github.com/tikv/tikv/issues/13313) @[glorv](https://github.com/glorv)
+
+  定时批量 DML 任务存在于一部分系统的业务逻辑中。 在这种场景下，批量写入任务会对在线交易的时延造成不小的影响。 在 v6.3.0 中，TiKV 对混合负载场景下读请求的优先级进行了优化。 在 v6.4.0 中， 利用同样的思路， TiKV 对写入请求也做了动态识别和动态优先级调整， 降低批量写入对交易事务写入的响应时间的影响。
```
Comment (qiancai) [inline releases/release-6.4.0.md:135]: 这里要提 v6.4 新增的 `apply-yield-write-size` 配置项吗？ @glorv 请帮忙看下，谢谢

File: releases/release-6.4.0.md [hunk 2]
```diff
@@ -130,6 +130,10 @@ TiDB 版本：6.4.0-DMR
 
     更多信息，请参考[用户文档](/system-variables.md#tidb_stats_load_sync_wait-从-v540-版本开始引入)。
 
+* 降低批量写入请求对轻量交易事务写入的响应时间的影响 [#13313](https://github.com/tikv/tikv/issues/13313) @[glorv](https://github.com/glorv)
```
Comment (qiancai) [inline releases/release-6.4.0.md:133]:
```suggestion
* 降低批量写入请求对轻量级事务写入的响应时间的影响 [#13313](https://github.com/tikv/tikv/issues/13313) @[glorv](https://github.com/glorv)
```

File: releases/release-6.4.0.md (hunk 1 above)
Comment (qiancai) [inline releases/release-6.4.0.md:135]:
```suggestion
  定时批量 DML 任务存在于一部分系统的业务逻辑中。 在此场景下，处理这些批量写入任务会对在线交易的时延造成不小的影响。在 v6.3.0 中，TiKV 对混合负载场景下读请求的优先级进行了优化，你可以通过 [`readpool.unified.auto-adjust-pool-size`](/tikv-configuration-file.md#auto-adjust-pool-size-从-v630-版本开始引入)配置开启对处理读请求的线程池的自动调整。在 v6.4.0 中， TiKV 对写入请求也进行了动态识别和优先级调整，你可以通过 [`apply-yield-write-size`](/tikv-configuration-file.md#apply-yield-write-size-从-v640-版本开始引入) 配置项控制 Apply 线程每一轮处理单个状态机写入的最大数据量，从而降低批量写入对交易事务写入的响应时间的影响。 
```

**Review by glorv — COMMENTED** [review]
LGTM

**Review by qiancai — APPROVED** [review]

File: releases/release-6.4.0.md [hunk 3]
```diff
@@ -130,6 +130,10 @@ TiDB 版本：6.4.0-DMR
 
     更多信息，请参考[用户文档](/system-variables.md#tidb_stats_load_sync_wait-从-v540-版本开始引入)。
 
+* 降低批量写入请求对轻量级事务写入的响应时间的影响 [#13313](https://github.com/tikv/tikv/issues/13313) @[glorv](https://github.com/glorv)
+
+  定时批量 DML 任务存在于一部分系统的业务逻辑中。在此场景下，处理这些批量写入任务会对在线交易的时延造成不小的影响。在 v6.3.0 中，TiKV 对混合负载场景下读请求的优先级进行了优化，你可以通过 [`readpool.unified.auto-adjust-pool-size`](/[tikv-configuration-file.md](http://tikv-configuration-file.md/)#auto-adjust-pool-size-从-v630-版本开始引入)配置项开启 TiKV 对处理读请求的线程池大小的自动调整。在 v6.4.0 中，TiKV 对写入请求也进行了动态识别和优先级调整，控制 Apply 线程每一轮处理单个状态机写入的最大数据量，从而降低批量写入对交易事务写入的响应时间的影响。
```
Comment (Oreoxmt) [inline releases/release-6.4.0.md:135]:
```suggestion
    定时批量 DML 任务存在于一部分系统的业务逻辑中。在此场景下，处理这些批量写入任务会增加在线交易的时延。在 v6.3.0 中，TiKV 对混合负载场景下读请求的优先级进行了优化，你可以通过 [`readpool.unified.auto-adjust-pool-size`](/tikv-configuration-file.md#auto-adjust-pool-size-从-v630-版本开始引入) 配置项开启 TiKV 对统一处理读请求的线程池 (UnifyReadPool) 大小的自动调整。在 v6.4.0 中，TiKV 对写入请求也进行了动态识别和优先级调整，控制 Apply 线程每一轮处理单个状态机写入的最大数据量，从而降低批量写入对交易事务写入的响应时间的影响。
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

**Pull request merged**